src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1479,20 +1475,41 @@ public void HandleCountChanges()
         {
             RemoteExecutor.Invoke(() =>
             {
-                Process p = Process.GetCurrentProcess();
-                int handleCount = p.HandleCount;
-                using (var fs1 = File.Open(GetTestFilePath(), FileMode.OpenOrCreate))
-                using (var fs2 = File.Open(GetTestFilePath(), FileMode.OpenOrCreate))
-                using (var fs3 = File.Open(GetTestFilePath(), FileMode.OpenOrCreate))
+                RetryHelper.Execute(maxAttempts: 10, () =>
```

Comment:
Why did you chose 10 cycles as the maximum? Why not 5, why not 15? Should we hardcode that as a const in the RetryHelper to have consistent retry numbers or vary between tests?

Reply:
Arbitrary.  I gave the argument a default value.